blazor/treeview/navigation.md

```diff
@@ -0,0 +1,332 @@
+</SfTreeView>
+</div>
+<div class="containers">
+    <p>Details of the clicked node</p>
+<table >
+        <tr>
+            <th style="width: 50%">Id</th>
+            <td style="width: 50%" align="center"><div>@Id</div></td>
+        </tr>
+        <tr>
+            <th style="width: 50%;">Text</th>
+            <td style="width: 30%" align="center">
+                <div>@Text</div>
+            </td>
+        </tr>
+        <tr><th style="width: 50%;" >Parent Id</th>
+            <td style="width: 30%" align="center">
+                    @if (ParentID == null)
+                    {
+                        <div>null</div>
+                    }
+                    else
+                    {
+                        <div>@ParentID</div>
+                    }
+            </td>
+        </tr>
+    </table>
+</div>
+@code {
+    SfTreeView<MailItem> tree;
+    public class MailItem
+    {
+        public string Id { get; set; }
+        public string ParentId { get; set; }
+        public string FolderName { get; set; }
+        public bool Expanded { get; set; }
+        public bool HasSubFolders { get; set; }
+        public Dictionary<string, object> HtmlAttributes { get; set; }
+    }
+
+    List<MailItem> MyFolder = new List<MailItem>();
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        MyFolder.Add(new MailItem
+            {
+                Id = "1",
+                FolderName = "Inbox",
+                HasSubFolders = true,
+                Expanded = true
+            });
+        MyFolder.Add(new MailItem
+            {
+                Id = "2",
+                ParentId = "1",
+                FolderName = "Categories",
+                Expanded = true,
+                HasSubFolders = true
+            });
+        MyFolder.Add(new MailItem
+            {
+                Id = "3",
+                ParentId = "2",
+                FolderName = "Primary"
+            });
+        MyFolder.Add(new MailItem
+            {
+                Id = "4",
+                ParentId = "2",
+                FolderName = "Social"
+            });
+        MyFolder.Add(new MailItem
+            {
+                Id = "5",
+                ParentId = "2",
+                FolderName = "Promotions"
+            });
+
+    }
+    public string Id;
+    public string Text;
+    public string ParentID;
+    public void nodeClicked(NodeClickEventArgs args)
+    {
+        Id = (args.NodeData.Id);
+        Text = (args.NodeData.Text);
+        ParentID = (args.NodeData.ParentID);
+    }
+}
+<style>
+    .containers {
+        margin-top: -200px;
+        margin-left: 500px;
+    }
+    .tree{
+        width:50%;
+    }
+</style>
+
+```
+
+## Tree node as hyperlink in Blazor TreeView Component
+

```

Comment:
The **NavigateUrl** property is used to navigate from one page to other pages on tree node selection. Using the CSS styles, you can customize the TreeView node as a hyperlink.

Reply:
Done.